cfme/tests/infrastructure/test_host.py

```diff
@@ -312,3 +312,41 @@ def _finalize():
             actual_state, power_state
         ),
     )
+
+
+@test_requirements.general_ui
+def test_infrastructure_hosts_refresh_multi(
+    appliance, setup_provider, provider
+):
+    """
+    Polarion:
+        assignee: prichard
+        casecomponent: Infra
+        caseimportance: low
+        initialEstimate: 1/3h
+        testSteps:
+            1. Navigate to the Compute > Infrastructure > Hosts view.
+            2. Select at least 2 hosts (more if possible) by checking the box in upper left of
```

Comment:
Optional: in the test I see that all hosts are refreshed but as it's clear what's going on, I leave it as optional